lib/MadNLPGPU/src/lapackgpu.jl

```diff
@@ -82,10 +82,10 @@ introduce(M::Solver) = "Lapack-GPU ($(M.opt.lapackgpu_algorithm))"
 if toolkit_version() >= v"11.3.1"
 
     is_inertia(M::Solver) = false # TODO: implement inertia(M::Solver) for BUNCHKAUFMAN
```

Comment:
I think it's totally relevant. It would be nice to have a KernelAbstraction kernel to compute the inertia on the GPU. The CPU algorithm is quite sequential in nature though. Maybe Magma is implementing its own kernel for inertia, I will investigate that. How about discussing that in a separate issue?

Reply:
Sounds good. If it is difficult to make it as a GPU kernel due to the sequential nature, we can probably just send `ipiv` to GPU

Reply:
Continuing the discussion here: 
https://github.com/sshin23/MadNLP.jl/issues/78